docs/api/resource.rst

```diff
@@ -351,10 +353,10 @@ GET /{collection}/<id>
 
 Returns a specific record by its id.
 
-For convenience and consistency, a header ``Last-Modified`` will also repeat the
+For convenience and consistency, a header ``ETag`` will also repeat the
 value of the ``last_modified`` field.
```

Comment:
Is there a reason to force server implementers to use milli-timestamp ETags? What would happen if someone implements a server that uses SHA hashes for ETags? This could easily happen if someone implements a server on top of some middleware library (like nodejs express or something), or configures a webserver like nginx to handle the whole ETag and 412/304 story.